packages/editor/src/components/assets/FileBrowser/FileBrowserGrid.tsx

```diff
@@ -178,7 +184,12 @@ type FileGridItemProps = {
 
 export const FileGridItem: React.FC<FileGridItemProps> = (props) => {
   const iconSize = useHookstate(getMutableState(FilesViewModeSettings).icons.iconSize).value
-  const staticResource = useFind(staticResourcePath, { query: { key: props.item.key } })
+  const staticResource = useFind(staticResourcePath, {
+    query: {
+      key: props.item.key,
+      project: getMutableState(EditorState).projectName.value!
```

Comment:
Make use of useMutateState hook and store projectName in a constant before using it here.

Reply:
done